frescobaldi_app/scorewiz/parts/special.py

```diff
@@ -30,6 +30,29 @@
 from . import register
 
 
+class Structure(_base.Part):
+    """Part type intended for laying out a piece's structure (breaks,
+    bar lines, etc.) separately from its musical content.
+
+    See the snippet titled "Using an extra voice for breaks" in the
+    Lilypond Notation Reference, section 4.3.1, for an example --
+    in that case, using an extra voice rather than a separate part.
+    """
+    @staticmethod
+    def title(_=_base.translate):
+        return _("Structure")
+
+    def build(self, data, builder):
+        p = ly.dom.ChordNames()
```

Comment:
I'm afraid he's not active anymore on Github. Jean, can you take it over?

Reply:
Hi all, thanks for your interest in these old patches. I have very limited time these days to commit to open source work, but am still interested in merging these in and will try to be more available while that's ongoing.

@jeanas, thanks for the information on using `Devnull` rather than `ChordNames`. I just committed that fix to my branch.